sale_procurement_amendment/__manifest__.py

```diff
@@ -0,0 +1,18 @@
+# -*- coding: utf-8 -*-
+# Copyright 2018 ACSONE SA/NV
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    'name': 'Sale Procurement Amendment',
+    'summary': """
+        Allow to reflect confirmed sale lines quantity amendments
+        to procurements""",
+    'version': '10.0.1.0.0',
+    'license': 'AGPL-3',
+    'author': 'ACSONE SA/NV,Odoo Community Association (OCA)',
+    'website': 'https://acsone.eu',
```

Comment:
@rousseldenis URL must be [this](https://github.com/OCA/maintainer-tools/blob/master/template/module/__manifest__.py#L11)

Reply:
Sure